### DISCUSSION TITLE
Jamulus server doest not record audio properly

### DISCUSSION THREAD

**Discussion by miro01001** [jamulussoftware/jamulus#1932] [Technical Support & Help] [answered]
Hi,
Recently updated to 3.8 and since then the server files does not record audio properly.
It does creates the appropriate files but audio files are all 0kb.
When opening the reaper files the tracks seem to be the right length but it is written OFFLINE all over them. 
Tried to switch back to 3.7 and does the same thing.
Please help.
Thank you

**Comment by dcorson-ticino-com** [comment, marked as answer]
Have you checked that the server still has free space to write the files?
That sounds like the disk is full.

**Reply by miro01001** [reply]
Yes indeed that was the problem!
thanks
